packages/icon-helpers/src/getAttributes.js

```diff
@@ -32,7 +33,13 @@ export default function getAttributes({
     attributes.title
   ) {
     iconAttributes.role = 'img';
-    iconAttributes.focusable = true;
+
+    // Reference:
+    // https://allyjs.io/tutorials/focusing-in-svg.html
+    if (tabIndex !== undefined && tabIndex !== null) {
+      iconAttributes.focusable = true;
+      iconAttributes.tabIndex = tabIndex;
```

Comment:
LGTM :+1: we may need to intercept this for Angular (and maybe Vue) to transform it to `tabindex` ... testing quickly it seems like it works camelCased so we may be fine

Reply:
Sounds good!